src/Chapter3.hs

```diff
@@ -970,6 +1078,23 @@ implement the following functions:
 
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
+data DayOfWeek = Mon | Tue | Wed | Thu | Fri | Sat | Sun deriving (Show, Eq, Ord, Enum)
+
+isWeekend :: DayOfWeek -> Bool 
+isWeekend = (>=Sat)
+
+nextDay :: DayOfWeek -> DayOfWeek
+nextDay day
+  | day >= Fri = Mon
+  | otherwise = succ day
```

Comment:
This function should return just the next day, not the "next working day". Otherwise, your implementation would be totally correct 👍🏻